app/assets/stylesheets/components/common/_newsletter_signup.scss

```diff
@@ -6,6 +6,11 @@
   @include body(16, 22);
 }
 
+.t-newsletter-strap {
```

Comment:
Ah right, in which case I think it was there initially as this was purely for A/B testing. As we're now implementing this as a real component, we can safely remove those prefixes.